src/toil_scripts/exome_variant_pipeline/README.md

```diff
@@ -26,6 +26,15 @@ or `pip install --pre toil-scripts` for cutting edge development version.
 
 Type: `toil-exome` to get basic help menu and instructions
 
+Our software lead prefers installation of python projects into a virtualenv. 
```

Comment:
Give me a break. This is not a mere preference of mine. It is a fact that not using virtualenvs increases the chance of versioning conflicts that can cause serious and hard to find bugs. I thought I had made that clear.